package.json

```diff
@@ -89,12 +99,24 @@
     "start-server-and-test": "^2.0.0",
     "styled-components": "^6.0.4",
     "supabase": "^1.142.2",
-    "typescript": "^5.2.2",
```

Comment:
Is there a reason we've moved this out of dependencies into devDependencies?

Reply:
there were conflicts in package.json when I rebase, and I accepted both comments, shall I add this so that the dependencies are not disrupted?

Reply:
I've moved it back to dependencies, no errors occurred